singletons/introspector.php

```diff
@@ -220,15 +220,15 @@ public function get_author_by_login($login) {
     ", $login));
     return $this->get_author_by_id($id);
   }
-  
+
   public function get_comments($post_id) {
     global $wpdb;
     $wp_comments = $wpdb->get_results($wpdb->prepare("
       SELECT *
       FROM $wpdb->comments
       WHERE comment_post_ID = %d
         AND comment_approved = 1
-        AND comment_type = ''
```

Comment:
This is not working with latest wordpress as default comment type is now "comment"